docs/developers/troy-overview.md

```diff
@@ -0,0 +1,39 @@
+# Troy - EVM on Celestia
+
+Developers can begin experimenting with solidity and vyper smart contracts
+using the new Troy testnet, an EVM compatible sovereign rollup.
+
+[Start building](./troy-smart-contract-tutorial.md)
+
+## About Troy testnet
+
+Previously, developers that wanted to experiment with smart contracts on
+Celestia’s Mamaki testnet had to spin up their own local sovereign rollup.
+For devs that also want to explore the process of deploying their own rollup,
+this step is not a problem. We recognize that there is a need for smart
+contract developers to have easier access while minimizing the friction
+for deploying on testnets.
+
+With the addition of the Troy testnet, smart contract developers will be
+relieved of unnecessary friction. Troy will serve as the place for developers
+to test and deploy Ethereum-style smart contracts without creating their own
+local network.
+
+Troy is an EVM compatible sovereign rollup built using Ethermint and Optimint.
```

Comment:
This is unnecessary. Instead, the "about troy" section should have something like
"Optimint and Ethermint, in tandem with the Cosmos SDK, serve as the basis for Troy. Optimint is an ABCI client for Cosmos SDK chains that enables them to become rollups. Ethermint is an EVM execution environment that plugs into the Cosmos SDK. Together, the combination creates an experimental EVM compatible sovereign rollup testnet."

Reply:
Got it, so replace the content under the "About Troy testnet" section with this?:

```
Optimint and Ethermint, in tandem with the Cosmos SDK, serve as the basis for Troy. Optimint is an ABCI client for Cosmos SDK chains that enables them to become rollups. Ethermint is an EVM execution environment that plugs into the Cosmos SDK. Together, the combination creates an experimental EVM compatible sovereign rollup testnet
```

Reply:
yeah

Reply:
Actually now that you mention, maybe it makes more sense to also put
"Any rollup testnets that are deployed on top of the Mamaki testnet are purely
for developer experimentation. At no point will they transition to mainnets.

In addition, Troy will run in “pessimistic” mode with a centralized sequencer
and without fraud proofs. Decentralized sequencers and fraud proofs are part
of our ongoing development process with Optimint."

from the mamaki testnet section in the about Troy section as well

Reply:
Ok got it, thank you for the help on this